opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/MetricPointAdapter.java

```diff
@@ -92,31 +98,23 @@ private boolean isNonMonotonic(MetricData metric) {
       double value,
       long epochNanos,
       long startEpochNanos) {
-    switch (metric.getType()) {
-      case NON_MONOTONIC_LONG:
-      case NON_MONOTONIC_DOUBLE:
-        return singleton(
-            new Gauge(metric.getName(), value, NANOSECONDS.toMillis(epochNanos), attributes));
-
-      case MONOTONIC_LONG:
-      case MONOTONIC_DOUBLE:
-        return singleton(
-            new Count(
-                metric.getName(),
-                value,
-                NANOSECONDS.toMillis(startEpochNanos),
-                NANOSECONDS.toMillis(epochNanos),
-                attributes));
-
-      default:
-        // maybe log something about unhandled types?
```

Comment:
Does this case no longer exist? The previous code returned an empty list of the type wasn't recognized.

Reply:
Yea, I went back and forth on this one a bit. I'm not convinced that such a case actually exists but I'll add the logic/comment back in.

Reply:
Done